src/app/development/middleware.js

```diff
@@ -64,6 +65,11 @@ async function templatesDisplayGet(req, res) {
     );
   }
 
+  if (templateId === "pyi-triage-desktop-download-app") {
+    renderOptions.qrCode =
+      await qrCodeHelper.generateQrCodeImageData("https://bbc.co.uk");
```

Comment:
Is there a way to test this as a standalone page? (can we not pass the `app-redirect` url here?)

Reply:
Do you mean the redirect route? I wasn't sure how/if that would work on `build` from the `all-templates` page. I was assuming that this would be covered later on once we have the routing in and some way of getting to these pages with a feature flag.
Also, I think on `build` the redirect URL will be read from the environment and so will be `get real value` at the moment.
We could update the template with some dummy values as long as we don't forget to update them later :)

Reply:
Yep the redirect route, I was hoping we could at least test the scanning of the QR code opening the app store link (even if it's just hardcoded for an iPhone for now)

Reply:
Ok, I've done that, we'll have to try it on `build` to see what actually happens - will the phone need to be on the VPN to access `build`?

Reply:
That's fine we can merge and test this in build given it's not live - phone shouldn't need to be on VPN